soundfile.py

```diff
@@ -441,6 +441,132 @@ def blocks(file, blocksize=None, overlap=0, frames=-1, start=0, stop=None,
                               dtype, always_2d, fill_value, out):
             yield block
 
+def append(data, file, samplerate=None, channels=None, format=None,
```

Comment:
I think `channels` has to be removed here.
It should be obtained from `data`, like in the `write()` function.